start.js

```diff
@@ -0,0 +1,55 @@
+const http = require('http');
+const soldiers = require('./routes/soldiers.js');
+const duties = require('./routes/duties.js');
+const justiceBoard = require('./routes/justiceBoard.js');
+
+function runServer() {
+  let server = http.createServer(function (req, res) {
+    let body = [];
+    req.on('data', (chunk) => {
+      body.push(chunk);
+    }).on('end', () => {
+      body = Buffer.concat(body).toString();
+      if (body !== "") {
+        body = JSON.parse(body);
+      }
+      const parsedUrl = (req.url.split("/"))[1];
+      if ("soldiers" === parsedUrl ||
+        (parsedUrl.length > 14 && "soldiers?name=" === parsedUrl.substr(0, 14))) {
```

Comment:
Impressive.
Notice that when you use numbers that don't have a clear meaning (0, 1) -
You should usually put them in a var so people know what you do.
So either put the number in a var, or the condition itself (condition is more fitting in this case)